test/integration/postgresql/serverspec/apache2_spec.rb

```diff
@@ -6,11 +6,6 @@
 end
 
 # Insert suite-specific tests below this line.
-describe file("#{$node['stash']['home_path']}/shared/stash-config.properties") do
-  it { should be_file }
-  it { should exist }
-end
-
```

Comment:
The stash-config.properties is now named bitbucket.properties. But this test was also in configuration_spec.rb and has nothing todo with apache. That's why I deleted it here.